src/SparqlHandler.js

```diff
@@ -33,18 +33,32 @@ export default class SparqlHandler {
       throw new Error(`${pathData} should at least contain a subject and a predicate`);
 
     // Create triple patterns
-    let queryVar = '?subject', sorts = [], clauses = [];
+    let queryVar = '?subject', sorts = [], clauses = [], languageFilters = [];
+    const closestLanguageRanges = [];
```

Comment:
We'll definitely need to modularize this in the future.

For now, could we use 1 or 2 helper methods within thie class to add the language constraints?